tests/acceptance/Makefile.am

```diff
@@ -21,6 +21,9 @@
 # (COSL) may apply to this file if you as a licensee so wish it. See
 # included file COSL.txt.
 #
+
+SUBDIRS = 25_cf-execd
```

Comment:
Because we build the `cf-execd-test` binary there (there is a `Makefile.am` added there in this PR as well). The other tests don't build anything, and hence don't need this.